providers/common/capabilities.c

```diff
@@ -30,7 +30,7 @@ typedef struct tls_group_constants_st {
     int maxdtls;             /* Maximum DTLS version (or 0 for undefined) */
 } TLS_GROUP_CONSTANTS;
 
-static const TLS_GROUP_CONSTANTS group_list[35] = {
+static const TLS_GROUP_CONSTANTS group_list[38] = {
```

Comment:
Is `38` even needed? Perhaps leave out the size and allow the compiler to determine it.